src/main/java/com/twilio/twiml/TwiML.java

```diff
@@ -126,8 +126,14 @@ public String toXml() throws TwiMLException {
             doc.appendChild(this.buildXmlElement(doc));
 
             TransformerFactory tFact = TransformerFactory.newInstance();
-            tFact.setAttribute(XMLConstants.ACCESS_EXTERNAL_DTD, "");
-            tFact.setAttribute(XMLConstants.ACCESS_EXTERNAL_STYLESHEET, "");
+
+            try {
+                tFact.setAttribute(XMLConstants.ACCESS_EXTERNAL_DTD, "");
+                tFact.setAttribute(XMLConstants.ACCESS_EXTERNAL_STYLESHEET, "");
+            } catch (IllegalArgumentException e){
+                //Unsupported Attributes
```

Comment:
Please add a log message to assist with debugging. Thank you!

Reply:
@thinkingserious what level would you want it raised as?

Reply:
@thinkingserious I went with debug as that is what is used elsewhere.